plugins/woocommerce-blocks/assets/js/blocks/checkout/inner-blocks/checkout-contact-information-block/block.tsx

```diff
@@ -11,54 +11,110 @@ import { ContactFormValues, getSetting } from '@woocommerce/settings';
 import {
 	StoreNoticesContainer,
 	CheckboxControl,
+	ValidatedTextInput,
 } from '@woocommerce/blocks-components';
 import { useDispatch, useSelect } from '@wordpress/data';
 import { CHECKOUT_STORE_KEY } from '@woocommerce/block-data';
 import { CONTACT_FORM_KEYS } from '@woocommerce/block-settings';
 import { Form } from '@woocommerce/base-components/cart-checkout';
 
-const Block = (): JSX.Element => {
-	const { customerId, shouldCreateAccount, additionalFields } = useSelect(
-		( select ) => {
-			const store = select( CHECKOUT_STORE_KEY );
-			return {
-				customerId: store.getCustomerId(),
-				shouldCreateAccount: store.getShouldCreateAccount(),
-				additionalFields: store.getAdditionalFields(),
-			};
-		}
+const CreateAccountUI = (): JSX.Element => {
```

Comment:
Same as other PR, this is deprecated now and since this is a new component we should update it.

```suggestion
const CreateAccountUI = (): React.ReactElement => {
```